gpkit/tests/t_small.py

```diff
@@ -69,9 +69,10 @@ def test_unitstr(self):
         x = gpkit.Variable("x", "ft")
         # pint issue 356
         footstrings = ("ft", "foot")  # backwards compatibility with pint 0.6
-        self.assertEqual(unitstr(gpkit.Variable("n", "count")), "count")
-        self.assertIn(unitstr(x), footstrings)
-        self.assertIn(unitstr(x.key), footstrings)
+        if gpkit.units:
+            self.assertEqual(unitstr(gpkit.Variable("n", "count")), "count")
+            self.assertIn(unitstr(x), footstrings)
+            self.assertIn(unitstr(x.key), footstrings)
```

Comment:
we used to, but since they're algebraically identical I wasn't sure that was the right behaviour anymore. Easy to change back!